src/config.rs

```diff
@@ -78,6 +78,7 @@ pub struct RobotConfig {
     pub variable_files: Vec<Utf8PathBuf>,
     pub argument_files: Vec<Utf8PathBuf>,
     pub exit_on_failure: bool,
+    pub secret_environment_variables: Vec<RobotFrameworkSecretEnvVar>,
```

Comment:
How can these be secret, if they are written to the config? Wouldn't it be more sensible to add them to `robot.yaml`? What exactly does secret mean here?

Reply:
Secret means that they are obfuscated in the Checkmk UI. But they are written to the config in plain text. Any agent plugin that needs a secret does the same. I am not aware of any better solution at the moment. Users can of course hard-code them in their test definitions if they want to.

Reply:
Ack